src/fidesops/ops/schemas/connection_configuration/connection_secrets_timescale.py

```diff
@@ -0,0 +1,12 @@
+from fidesops.ops.schemas.base_class import NoValidationSchema
+from fidesops.ops.schemas.connection_configuration.connection_secrets_postgres import (
+    PostgreSQLSchema,
+)
+
+
+class TimescaleSchema(PostgreSQLSchema):
```

Comment:
`PostgresSQLSchema` because timescale is built on that and you are just extending it?

Reply:
exactly yes, I'll add a code comment to make this clearer. I could have used the PostgreSQLSchema and PostgreSQLConnector themselves below directly but I assumed at some point the two might diverge?